pgptool-gui/src/main/java/org/pgptool/gui/app/EntryPoint.java

```diff
@@ -220,7 +220,7 @@ private static void reportAppInitFailureMessageToUser(Window window, Throwable t
 		String msg = ConsoleExceptionUtils.getAllMessages(t);
 		msg += "\r\n" + versionInfo;
 		UiUtils.messageBox(UiUtils.actionEvent(window, "reportAppInitFailureMessageToUser"), msg,
-				"PGP Tool startup failed" + versionInfo, JOptionPane.ERROR_MESSAGE);
+				"PGPTool startup failed" + versionInfo, JOptionPane.ERROR_MESSAGE);
```

Comment:
It's important to be consistent with the name.  However, throughout the app (and on www.pgptool.github.io), there are some places where the app name is shown as **_PGPTool_** and in other places _**PGP Tool**_ (with a space between "PGP" and "Tool")
**_PGPTool_** may be the preferred name to use because it's easier in searches if the app name is one word

Reply:
I agree -- consistency is a very good thing. 
For the purpose of SEO I think "PGP Tool" is a better option.